src/test/scala/akka/persistence/cassandra/query/CassandraReadJournalIntegrationSpec.scala

```diff
@@ -180,18 +217,42 @@ class CassandraReadJournalIntegrationSpec extends TestKit(ActorSystem("test", co
         received.expectMsgType[EventEnvelope]
         received.expectMsgType[EventEnvelope]
         Thread.sleep(6000) // poll interval + 1
-        
-        system.stop(doc)
-        journal.shutdown().futureValue        
       }
+    }
 
-      /*
-      "discover real-time events as they are added, across partition boundaries" in {
-        pending
+    "answering a query with offset" should {
+      "return only events from time windows at or after the offset" in fixture { f =>
+        import f._
+        
+        def send(i:Int) = { 
+          doc ! s"change-$i"
+          probe.expectMsgType[String] // we know it's persisted once it hits the probe          
+        }
+        
+        send(1)
+        Thread.sleep(1000) // time window is 1 second
+        send(2)
+        Thread.sleep(1000) // time window is 1 second
+        send(3)
+        Thread.sleep(1000) // time window is 1 second
+        send(4)
+        val offset = System.currentTimeMillis()
```

Comment:
Would it not make more sense to move this line to before `send(4)` ?
If the method send takes more time than the time window, then isn't there a chance that it will not be the first event seen?